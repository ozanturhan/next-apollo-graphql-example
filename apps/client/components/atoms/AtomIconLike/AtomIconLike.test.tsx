import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import { AtomIconLike } from './index';

it('renders correctly', () => {
  const tree = render(<AtomIconLike />).container;
  expect(tree).toMatchSnapshot();
});
