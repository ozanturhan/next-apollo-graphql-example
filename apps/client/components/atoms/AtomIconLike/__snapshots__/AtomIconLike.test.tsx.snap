// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  height: 18px;
}

<div>
  <svg
    class="c0"
    viewBox="0 0 391.837 391.837"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M285.257 35.528c58.743.286 106.294 47.836 106.58 106.58 0 107.624-195.918 214.204-195.918 214.204S0 248.165 0 142.108c0-58.862 47.717-106.58 106.58-106.58a105.534 105.534 0 0 1 89.339 48.065 106.578 106.578 0 0 1 89.338-48.065z"
      fill="#d7443e"
    />
  </svg>
</div>
`;
